core/build/src/lib.rs

```diff
@@ -168,26 +168,28 @@ impl ParseCallbacks for SgxParseCallbacks {
     }
 
     fn add_derives(&self, name: &str) -> Vec<String> {
-        if self.dynamically_sized_types.iter().any(|n| *n == name) {
-            // For dynamically sized types we don't even derive Debug, because
-            // they are often times packed and packed types can't derive Debug
-            // without deriving Copy, however by the dynamic nature one can't
-            // derive Copy
-            return vec![];
-        }
-        let mut attributes = vec!["Debug"];
+        let mut attributes = vec![];
```

Comment:
This is just a comment not a request for changes but in general I think it's better to make changes to the build utilities in a separate commit if not a separate pull request, even if they're motivated by the "main" contents of the PR.

Reply:
That sounds reasonable. With the stacking provided by graphite it should be easier to break them out and stack them appropriately.

Reply:
If you do split this out, could you make all these tests use `.contains()` instead of `.iter().any()`? That feels like something a future clippy will complain about :-)